### ISSUE TITLE
I'm sorry to bother you again.

### ISSUE THREAD

**Issue by yjydiancom** [dcaffo98/path-planning-cnn#1]
![image](https://user-images.githubusercontent.com/91858119/223028768-6c9a0040-c6a4-4f61-977f-f688dc3ddf62.png)
![image](https://user-images.githubusercontent.com/91858119/223029158-2fe2fafc-6fec-4a5d-b9fc-35aa8273717b.png)
I'm sorry I'm only getting back to you now, as training is time consuming and there are other jobs. After seeing your previous reply to my comment, I trained again myself and the results were still bad. It can be seen that even by the 100th round of training, the loss stops decreasing when it reaches around 0.00018 and the predicted graph is very bad. I used the MSE. Can you share your trained checkpoints file?

**Comment by yjydiancom** [comment]
Thank you for your continued patience in replying to me and I would appreciate it if I could receive a reply from you this time.

**Comment by dcaffo98** [comment]
This is the [checkpoint](https://drive.google.com/file/d/13YPPRkzAMDDencW3JAEjLonLUi19lL1L/view?usp=share_link) whose results are shown on Medium. Hope you'll be able to reproduce them as well. The minimum loss is around 0.00003. Anyway, I see you're training for a lot of epochs. How many samples are in your training dataset?

**Comment by yjydiancom** [comment]
> 这是检查[点](https://drive.google.com/file/d/13YPPRkzAMDDencW3JAEjLonLUi19lL1L/view?usp=share_link)，其结果显示在“中”上。希望您也能够重现它们。最小损失约为0.00003。无论如何，我看到你正在训练很多时代。训练数据集中有多少个样本？

OK, thanks! I'll give it a try! I don't seem to have put in the full training dataset you gave.
